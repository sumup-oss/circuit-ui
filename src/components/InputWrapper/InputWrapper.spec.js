import React from 'react';

import { InputWrapper } from '.';

describe('InputWrapper', () => {
  it("should have it's base styles", () => {
    const actual = create(<InputWrapper />);
    expect(actual).toMatchSnapshot();
  });
});
