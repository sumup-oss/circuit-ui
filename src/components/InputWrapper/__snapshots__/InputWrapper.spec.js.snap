// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`InputWrapper should have it's base styles 1`] = `
.circuit-0 {
  display: inline-block;
  position: relative;
}

<div
  className="circuit-0"
/>
`;
